src/rebar_git_resource.erl

```diff
@@ -274,19 +265,6 @@ make_vsn_(Dir) ->
 
 %% Internal functions
 
-git_ref(Dir, Arg) ->
```

Comment:
This is a consequence of https://github.com/erlang/rebar3/pull/2412/files#r514646991.